packages/terra-image/src/Image.jsx

```diff
@@ -23,6 +23,10 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string.isRequired,
+  /**
+   * The source string for the image which will be displayed during loading and in case of src load failure.
+   */
+  placeholderSrc: PropTypes.node,
```

Comment:
Sorry, I should have been more clear. I'd like this to take a react element instead of a source string and just render the react element.

Reply:
ah, okay, my mistake. updates have been pushed